pkg/operator/controller/controller_lb.go

```diff
@@ -51,12 +50,6 @@ func (r *reconciler) ensureLoadBalancerService(ci *operatorv1.IngressController,
 	return currentLBService, nil
 }
 
-// TODO: This should take operator config into account so that the operand
-// namespace isn't hard-coded.
-func loadBalancerServiceName(ci *operatorv1.IngressController) types.NamespacedName {
```

Comment:
All other namespaced name functions live in `names.go`. Should `loadBalancerServiceName` do the same?